2023/README.md

```diff
@@ -102,15 +101,20 @@ show that specific test case among various of testruns, `show [test_run_id];` wi
 **Mentors**: Charan Kamarapu, Ritik Jain, Shubham Jain.
 
 **Goals & Ideas**
-* Add support for dependencies in typescript.
-* 
+* Add support for `Httpclients` in typescript. 
+    - This sdk currently supports `node-fetch` httpClient, to support other httpclients like `axios`, `http`, etc. You can take the 1st ref below which intercepts other popular httpClients.
+* Add support for popular `Sql` databases. For eg: `PostgreSQL`, `MySQL`, etc. By adding support for `Sequelize`.
+* Add support for `MongoDB` framework.
+
```

Comment:
Needs more framework, task is too short for even 175 hours.

Reply:
Add support for Amazon S3, Dynamodb, Azure blob storage, and Google Cloud Storage using the HTTP Client. 
Add support for Redis and elastic-search.